OPERATING.md

```diff
@@ -0,0 +1,79 @@
+# Operating a Convene Neighborhood
+
+- [Social Responsibilities](#social-responsibilities)
+  - [As a Cooperative](#as-a-cooperative)
+  - [As a Person](#as-a-person)
+  - [As a Legacy Organization](#as-a-legacy-organization)
+- [Economic Model](#economic-model)
+  - [As a Cooperative](#as-a-cooperative-1)
+  - [As a Person](#as-a-person-1)
+  - [As a Classic Corporation](#as-a-classic-corporation)
+- [Technology Details](#technology-details)
+  - [As a Cooperative](#as-a-cooperative-2)
+  - [As a Person](#as-a-person-2)
+  - [As a Classic Non-Profit, Government, or Corporation](#as-a-classic-non-profit-government-or-corporation)
+
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
```

Comment:
I agree this is a `stream of consciousness` and IMHO a bit to vague to easily understand.

Can the purpose/value of convene be explained in just 1 - 2 sentences in laymans/simple terms?

E.g. Convene provides a digital infrastructure space and tools that help small businesses/non-profits/grassroots orgs operate their businesses cheaper than enterprise vendors.

Reply:
That is what the README is for.